src/adjustViewportsWithContexts.js

```diff
@@ -21,10 +21,10 @@ module.exports = function adjustViewportsWithContexts(contexts, opt) {
     maxViewport: contextMaxViewport,
   }
 
-  if (opt.minViewport !== undefined) {
+  if (opt.minViewport) {
```

Comment:
`0` will be falsy

Reply:
> Do you really think 0 would make sense for a viewport width?
No, but i think user can try it. But It should’t fail to run, it will take the default value.